### PR TITLE
UserId module to export user ids as eids with pbjs.getUserIdsAsEids()

### PR DESCRIPTION
**Pull request by pm-harshad-mane**
## Type of change
- [X] Feature

## Description of change
UserId module to export user ids as eids with pbjs.getUserIdsAsEids()

### CODE REVIEW

**Comment by jsnellbaker** [comment]
@pm-harshad-mane Do we need a docs PR to list this new function on the Prebid API?

File: modules/userId/index.js [hunk 1]
```diff
@@ -576,6 +586,7 @@ export function init(config) {
 
   // exposing getUserIds function in global-name-space so that userIds stored in Prebid can be used by external codes.
   (getGlobal()).getUserIds = getUserIds;
+  (getGlobal()).getUserIdsAsEids = getUserIdsAsEids
```
Comment (idettman) [inline modules/userId/index.js:589]: we should be consistent with the semicolon line endings, so please add one to line 589

Comment (pm-harshad-mane) [reply modules/userId/index.js]: Added the missing semicolon :)

**Comment by pm-harshad-mane** [comment]
Hello @idettman ,
I will raise a separate documentation PR to prebid.github.io
Can you please merge this?

**Comment by pm-harshad-mane** [comment]
Hello @idettman ,
The CI is passing, documentation is also added.
Could you please merge the PR?

**Pull request merged**